tests/ludwig/utils/test_trainer_utils.py

```diff
@@ -61,6 +62,7 @@ def test_progress_tracker():
         "num_increases_bs": 0,
         "num_reductions_lr": 0,
         "steps": 0,
+        "tune_ckpts": 0,
```

Comment:
nit: Use the full name, `tune_checkpoints`.

Reply:
Done.  Changed name to tune_checkpoint_num as per your suggestion in slack.